src/pages/BillsReport.vue

```diff
@@ -128,6 +190,17 @@ export default {
       ],
       prev: 750,
       current: 711.48,
+      dateBR: {
+        days: ["Domingo", "Segunda", "Terça", "Quarta", "Quinta", "Sexta", "Sábado"],
+        daysShort: ["Dom", "Seg", "Ter", "Qua", "Qui", "Sex", "Sáb"],
+        months: ["Janeiro", "Fevereiro", "Março", "Abril", "Maio", "Junho", "Julho", "Agosto", "Setembro", "Outubro", "Novembro", "Dezembro"],
+        monthsShort: ["Jan", "Fev", "Mar", "Abr", "Mai", "Jun", "Jul", "Ago", "Set", "Out", "Nov", "Dez"],
+        firstDayOfWeek: 1
+      },
```

Comment:
Doesn't need to be done, but date1 and date2 is a way of seeing that you didn't componentize something